File: cases/framework_provider_taxonomy/positive.ts
import { LlmAgent } from '@google/adk';
import { Kernel } from '@microsoft/semantic-kernel';
import { VectorStoreIndex } from '@llamaindex/core';
import { Agent } from '@mastra/core/agent';
import { GoogleGenAI } from '@google/genai';
import { google } from '@ai-sdk/google';
import { BedrockRuntimeClient } from '@aws-sdk/client-bedrock-runtime';

const agent = new Agent({ model: 'gemini-2.5-flash' });

import { ToolLoopAgent } from 'ai';
import type { ModelMessage } from 'ai/internal';

File: cases/framework_provider_taxonomy/positive.py
import agno.agent
import google.adk
import llama_index.core
import semantic_kernel
import smolagents
from google import genai
from google.genai import types
from langchain_aws import ChatBedrock

import boto3


bedrock = boto3.client("bedrock-runtime")
agent = google.adk.Agent(model="gemini-2.5-pro")

